Makefile

```diff
@@ -36,9 +36,13 @@ help: ## Display this help.
 
 ##@ Development
 
+.PHONY: clean-bin
+clean-bin: ## Clean external tools
+	@rm -rf ${PROJECT_BIN}
+
 .PHONY: clean
 clean: ## Clean external tools and output dirs
-	@rm -rf ${PROJECT_BIN} ${PROJECT_DIST} ${GENERATED_CLIENT_PATH}/api ${GENERATED_CLIENT_PATH}/models ${GENERATED_CLIENT_PATH}/horreum_raw_client.py ${GENERATED_CLIENT_PATH}/kiota-lock.json
+	@rm -rf ${PROJECT_DIST} ${GENERATED_CLIENT_PATH}/api ${GENERATED_CLIENT_PATH}/models ${GENERATED_CLIENT_PATH}/horreum_raw_client.py ${GENERATED_CLIENT_PATH}/kiota-lock.json
```

Comment:
I need to preserve the `src/horreum/raw_client/__init__.py` and `src/horreum/raw_client/.gitignore` otherwise the imports of the generated clients would fail.

> do you need to explicitly create ${GENERATED_CLIENT_PATH} or does kiota do that implicitly?

No need to explicitly create it, but I would keep that folder instead of removing it during the `clean`

Reply:
nvm I am going to change the imports such that I do not need the `__init__.py`, in this case I agree with you I can completely remove the `${GENERATED_CLIENT_PATH}` folder.